base/Command.js

```diff
@@ -64,14 +64,21 @@ class Command {
     return `${line1}${line2}`;
   }
 
+  async embedMessage(message) {
+    if (!message) throw new this.client.methods.errors.ParseError("Invalid format, missing required value: MESSAGE");
+    const embed = await new MessageEmbed().setColor(message.guild.me.roles.highest.color || 5198940).setTimestamp().setFooter(`Requested by ${message.member.user.username}`, message.guild.me.user.displayAvatarURL());
```

Comment:
why await?

Reply:
You have to create the embed...

Reply:
This is pointless since JS doesn't have async constructors aswell as non of the methods return a Promise

Reply:
^ exactly